bin/rtlcss.js

```diff
@@ -197,8 +205,9 @@ if (!shouldBreak) {
               }
             })
           } else {
-            // process only *.css
-            if (/\.(css)$/.test(file)) {
+            // process only *.css and ignore RTLed files
+            var patt = new RegExp('('+ext+')', 'gm');
```

Comment:
This regex does not escape the input string, for example, `style.css` would also match `stylercss.css`!